modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/quantity_selector/InputQuantitySelector.js

```diff
@@ -61,14 +76,104 @@ const InputQuantitySelector = forwardRef(
 			}, 500)
 		);
 
+		// eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Are you sure this is needed? I don't see dependencies here

Reply:
These disable-next-line it's here because the following method should be a dependency of an useEffect but putting the instruction on it doesn't remove the warning, only putting it here does (and it's the correct "type" of disable)